### ISSUE TITLE
vectorized dp optimizer

### ISSUE THREAD

**Issue by dionman** [tensorflow/privacy#75]
What's the difference between vectorized and older versions of dp optimizers? Are vectorized optimizers tested to the same extent?

**Comment by npapernot** [comment]
The optimizers should produce the same results, the vectorized version of the optimizer improves wall-clock training time by parallelizing the per-example gradient computation using [tf.vectorized_map](https://www.tensorflow.org/api_docs/python/tf/vectorized_map) at the expense of a larger memory footprint.
